src/ngraph_encapsulate_op.cc

```diff
@@ -258,28 +258,10 @@ class NGraphEncapsulateOp : public OpKernel {
     return Status::OK();
   }
 
-  //---------------------------------------------------------------------------
-  // OpKernel::Compute
-  //---------------------------------------------------------------------------
-  void Compute(OpKernelContext* ctx) override {
-    std::ostringstream oss;
-    oss << "Execute: Encapsulate_" << my_instance_id << ": " << name();
-    ngraph::Event event(oss.str(), name(), "");
-
-    Timer compute_time;
-    std::lock_guard<std::mutex> lock(m_compute_lock);
-    NGRAPH_VLOG(4) << "NGraphEncapsulateOp::Compute starting for cluster "
-                   << m_ngraph_cluster;
-
-    NGRAPH_VLOG(4) << "Got backend of type: " << m_op_backend_name;
-    ng::runtime::Backend* op_backend =
-        BackendManager::GetBackend(m_op_backend_name);
-
-    ngraph::Event event_func_maybe_create("FunctionMaybeCreate", name(), "");
-    Timer function_lookup_or_create;
+  Status ComputeSignature(OpKernelContext* ctx, std::stringstream& signature_ss,
```

Comment:
`ComputeSignature` vs `get_ng_exec`. I think we use `snake_case` in python and `UpperCamelCase` in cpp. Can you please also rename `get_current_ng_tensor` to `UpperCamelCase`.